### PR TITLE
global: upgrade to 12.0.0b3.dev7

### PR DESCRIPTION
**Pull request by zzacharo**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -81,3 +81,6 @@ celerybeat-schedule.db
 # node modules
 **/*/node_modules
 node_modules
+
+# local tmp folder
+tmp
```
Comment (kpsherva) [inline .gitignore:86]: why is tmp needed?

Comment (zzacharo) [reply .gitignore:86]: not needed I added so I can keep track of some local files without committing them. I can remove them if you think it is confusing.

File: site/setup.cfg [hunk 2]
```diff
@@ -20,7 +20,7 @@ tests =
 [options.entry_points]
 flask.commands =
     cds-admin = cds_rdm.cli:cds_admin
-    migration = cds_rdm.migration.cli:migration
+    # migration = cds_rdm.migration.cli:migration
```
Comment (kpsherva) [inline site/setup.cfg:23]: nit: maybe it is better to group the commented out entrypoints together

File: Pipfile [hunk 3]
```diff
@@ -22,5 +24,6 @@ uwsgi-tools = ">=1.1.1"
 [requires]
 python_version = "3.9"
 
+# disable allow_prereleases when we upgrade invenio-rdm-migrator
 [pipenv]
-allow_prereleases = false
+allow_prereleases = true
```
Comment (kpsherva) [inline Pipfile:29]: do we need pre-releases?

Comment (ntarocco) [reply Pipfile:29]: Please no! :)

File: invenio.cfg [hunk 4]
```diff
@@ -329,3 +330,7 @@ CDS_EOS_OFFLOAD_ENABLED = False
 CDS_LOCAL_OFFLOAD_ENABLED = False
 CDS_LOCAL_OFFLOAD_FILES = ["file.txt", "file2.txt"]
 CDS_LOCAL_OFFLOAD_STORAGE = ""
+
+# CDS Administration panel
+# ========================
+USERS_RESOURCES_ADMINISTRATION_ENABLED = True
```
Comment (ntarocco) [inline invenio.cfg:336]: I think we really need to change `invenio-users-resources` and now enable it by default. Feel free to add a commit there

**Review by ntarocco — APPROVED** [review]

**Review by ntarocco — APPROVED** [review]

**Pull request merged**